### PR TITLE
add WebGL 2.0 to status list

### PR DESCRIPTION
**Pull request by patrickkettner**

### CODE REVIEW

File: app/static/ie-status.json [hunk 1]
```diff
@@ -1885,6 +1885,24 @@
...
+      "iePrefixed": "",
+      "ieUnprefixed": "",
+      "priority": "medium"
+    },
+    "msdn": "",
+    "wpd": "",
+    "demo": "https://www.youtube.com/watch?v=2v6iLpY7j5M",
+    "id": 6694359164518400 ,
+    "uservoiceid": 6517037
+  }
```
Comment (petetnt) [inline app/static/ie-status.json]: Missing a comma here :hamburger:

Comment (jacobrossi) [inline app/static/ie-status.json]: Needs a comma

File: app/static/ie-status.json [hunk 2]
```diff
@@ -1885,6 +1885,24 @@
     "id": 6049512976023552
   },
   {
+    "name": "WebGL 2.0",
+    "category": "Graphics",
+    "link": "https://www.khronos.org/registry/webgl/specs/latest/2.0/",
+    "summary": "An update to WebGL, based on OpenGL ES 3.0",
+    "standardStatus": "Working draft or equivalent",
+    "ieStatus": {
+      "text": "Under Consideration ",
```
Comment (molant) [inline app/static/ie-status.json]: extra space at the end of the string

File: app/static/ie-status.json [hunk 3]
```diff
@@ -1885,6 +1885,24 @@
...
+      "iePrefixed": "",
+      "ieUnprefixed": "",
+      "priority": "medium"
+    },
+    "msdn": "",
+    "wpd": "",
+    "demo": "https://www.youtube.com/watch?v=2v6iLpY7j5M",
+    "id": 6694359164518400 ,
```
Comment (molant) [inline app/static/ie-status.json]: extra space after the number

File: app/static/ie-status.json [hunk 4]
```diff
@@ -1885,6 +1885,24 @@
...
+    "standardStatus": "Working draft or equivalent",
+    "ieStatus": {
+      "text": "Under Consideration ",
+      "iePrefixed": "",
+      "ieUnprefixed": "",
+      "priority": "medium"
+    },
+    "msdn": "",
+    "wpd": "",
+    "demo": "https://www.youtube.com/watch?v=2v6iLpY7j5M",
```
Comment (molant) [inline app/static/ie-status.json:1901]: Not sure we want to link the youtube video. This field is to link to demos hosted in http://dev.modern.ie/testdrive/ (although we haven't used it yet).
@jacobrossi what do you think?
In any case, cool stuff!

Comment (patrickkettner) [inline app/static/ie-status.json:1901]: I noticed there was [another link to a youtube video](https://github.com/patrickkettner/Status/blob/27496c1ff9544a42acf716d3b4784558ecaaf741/app/static/ie-status.json#L1901), for what thats worth

Comment (jacobrossi) [inline app/static/ie-status.json:1901]: The demo links don't appear in the UI. But I see no big deal with a Youtube link in lieu of having a test drive demo (esp for stuff we don't support yet)

**Comment by molant** [comment]
@jacobrossi @patrickkettner quick question, what is the difference between this and the one in [here](https://github.com/patrickkettner/Status/blob/add-webgl2/app/static/ie-status.json#L4084)

**Comment by jacobrossi** [comment]
Looks like there was an old mislabeled one, which we missed (oops, sorry).  But @patrickkettner 's entry is much more detailed. So I deleted the old one when I merged in https://github.com/MicrosoftEdge/Status/commit/4b16f71b13d192cb4204e41d4eeba1f420673e72

**Pull request closed without merging**